Doc/whatsnew/3.14.rst

```diff
@@ -2501,6 +2505,11 @@ Changes in the Python API
   See :ref:`above <whatsnew314-typing-union>` for more details.
   (Contributed by Jelle Zijlstra in :gh:`105499`.)
 
+* The behavior of annotations has changed in various ways; see
```

Comment:
```suggestion
* The runtime behavior of annotations has changed in various ways; see
```
Might be good to specify these aren't typing changes but rather runtime changes? Up to you!

Reply:
Good point, that's worth emphasizing. Static typing is basically completely unaffected.